npc/namespace.go

```diff
@@ -471,7 +471,19 @@ func (ns *ns) deleteNamespace(obj *coreapi.Namespace) error {
 	}
 
 	// Remove namespace ipset from any matching namespace selectors
-	return ns.nsSelectors.delFromMatching(obj.ObjectMeta.UID, obj.ObjectMeta.Labels, string(ns.allPods.ipsetName))
+	err := ns.nsSelectors.delFromMatching(obj.ObjectMeta.UID, obj.ObjectMeta.Labels, string(ns.allPods.ipsetName))
+	if err != nil {
+		return err
+	}
+
+	// Flush defaultAllowIPSet
+	err = ns.ips.Flush(ns.defaultAllowIPSet)
```

Comment:
Flush is redundant here, as the ipset is of the hash:ip type, thus Destroy will delete it even if it contains any entry.

Reply:
I will remove it